src/main/java/uk/gov/ons/census/casesvc/service/CaseReceipter.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.ons.census.casesvc.service;
+
+import static uk.gov.ons.census.casesvc.utility.QuestionnaireTypeHelper.iscontinuationQuestionnaireTypes;
+
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.casesvc.model.entity.Case;
+import uk.gov.ons.census.casesvc.model.entity.UacQidLink;
+
+@Component
+public class CaseReceipter {
+  private final CaseService caseService;
+
+  public CaseReceipter(CaseService caseService) {
+    this.caseService = caseService;
+  }
+
+  public void handleReceipting(Case caze, UacQidLink uacQidLink) {
+    if (uacQidLink.isActive()) return;
```

Comment:
I presume this check is because the flow we want is to always deactivate the UAC before calling this function, but if we're doing that think it should actively throw an exception instead of just returning if that's not the followed, maybe throw an `IllegalArgumentException`?

Reply:
Good spot, I think this should be moved back to the Linking Service where it might not be inactive.  Making it inactive is what the receiptService does, so silly to test for it further along